lib/preprocessors/glimmer.js

```diff
@@ -5,9 +5,11 @@ const {
   preprocessEmbeddedTemplates,
 } = require('ember-template-imports/lib/preprocess-embedded-templates');
 const util = require('ember-template-imports/src/util');
+const DocumentLines = require('../utils/document');
 
 const TRANSFORM_CACHE = new Map();
 const TEXT_CACHE = new Map();
+const RULE_CACHE = new Map();
```

Comment:
will this be used in a future PR?

Reply:
no, it can be removed, I do it once https://github.com/ember-cli/eslint-plugin-ember/pull/1852 is merged